tools/requirements/docs.txt

```diff
@@ -1,5 +1,7 @@
 sphinx == 3.2.1
 furo
+myst_parser
```

Comment:
The compulsive one inside me wants a canonical name.

Reply:
Will change in the follow up! :)